################################################################
################################################################
#                                                              #
#             .yaml for Neobotix USBoard                       #
#                                                              #
################################################################
################################################################

##########################
#    device parameter    #
##########################
ComPort: /dev/ttyUSB0
usboard_timeout_: 0.5
requestRate: 5
log: false

##########################
#        sensors         #
##########################
sensor1_active: true
sensor2_active: true
sensor3_active: true
sensor4_active: true
sensor5_active: true
sensor6_active: true
sensor7_active: false
sensor8_active: false
sensor9_active: false
sensor10_active: false
sensor11_active: false
sensor12_active: false
sensor13_active: false
sensor14_active: false
sensor15_active: false
sensor16_active: false
